### DISCUSSION TITLE
Unable to pin public repo that I was a major contributor to

### DISCUSSION THREAD

**Discussion by robertpuccinelli** [community/community#52474] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

There is a public repo that I was a significant contributor to and I would like to advertise it on my profile with a pin. However, all my attempts to pin it have failed, so I'm wondering if there is an org setting that prevents it from being pinned or if this specific repo is bugged out.

When I go to the repo, click "Edit Pins" and then "Pin to... Profile", I get a banner message stating that my pins have been updated. However, when I visit my profile, I see that none of my pins have changed even though there is space for more. 

I do not see this repo listed when I click "Customize your pins" on my profile page and it does not appear when I search for the name in the "Top Repositories" list on the main home page. Regarding "Top Repositories", I noticed that the only org repos that appear are private and belong to the team that I contributed to.

Thanks!

**Comment by robertpuccinelli** [comment]
After I starred the repo, I was able to pin it. Seems like a bug to me, but I was able to resolve my own issue.

**Reply by tomas2870** [reply]
I'm currently having the same issue. After I starred it, nothing changed. Did you do anything else?

**Comment by despargy** [comment]
I do have the same issue. I am unable to pin the repo that my colleague invited me as a collaborator. 
I have commit/push many times, I have checked the linked email to match those from my git account, I have starred the repo (undo and do many times) and also I have updated the git version using Ubuntu 20.04 and then pushed from the new git version (2.49.0) too.

Please, if someone knows provide the solution.
